src/main.py

```diff
@@ -179,6 +179,12 @@ async def on_message(message: discord.Message):
         log_info = parse_log(log_link)
         await message.channel.send(log_info)
 
+    for community_wiki_link in re.findall(r"https://stardewcommunitywiki.com/[a-zA-Z0-9_/:-%]+", message.content):
+        link_path = urllib.parse.urlparse(community_wiki_link).path
+        new_url = "https://stardewvalleywiki.com" + link_path
```

Comment:
I think we would want a `/` between the hardcoded URL and `link_path`. There's a `urllib.urljoin` which would be good here.